src/providers/oracle/qgsoracleprovider.cpp

```diff
@@ -44,11 +45,16 @@
 #endif
 
 #include <QSqlRecord>
+#include <QSqlDriver>
 #include <QSqlField>
 #include <QUuid>
 
 #include "ocispatial/wkbptr.h"
 
+
+#define LoggedExecStatic(query, sql, args, uri, _class) execLoggedStatic( query, sql, args, uri, _class, QString(QString( __FILE__ ).mid( sOracleConQueryLogFilePrefixLength ) + ':' + QString::number( __LINE__ ) + " (" + __FUNCTION__ + ")") )
```

Comment:
_class is always QgsOracleProvider in this file, maybe we can remove it as a parameter no?

Reply:
done